libs/generic-view/models/src/lib/modal-visibility-controller.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { z } from "zod"
+
+const dataValidator = z
+  .object({
+    visible: z.boolean().optional().default(true),
```

Comment:
if you apply default value it's no longer optional :)

Reply:
Changed in [`522ee44` (#1863)](https://github.com/mudita/mudita-center/pull/1863/commits/522ee449b040eb751c7ee4c56c4a57adb496659a)